dss/events/handlers/index.py

```diff
@@ -1,21 +1,25 @@
+
+
```

Comment:
Lines 158 and 159 are two blank lines between top level functions, the next of which starts on line 160. This is consistent with PEP 8, which I thought we were using.